config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_discoveryenginedatastores.discoveryengine.cnrm.cloud.google.com.yaml

```diff
@@ -126,37 +243,42 @@ spec:
                 type: string
               solutionTypes:
                 description: |-
+                  Immutable.
                   The solutions that the data store enrolls. Available solutions for each
-                  [industry_vertical][google.cloud.discoveryengine.v1.DataStore.industry_vertical]:
+                   [industry_vertical][google.cloud.discoveryengine.v1alpha.DataStore.industry_vertical]:
 
-                  * `MEDIA`: `SOLUTION_TYPE_RECOMMENDATION` and `SOLUTION_TYPE_SEARCH`.
-                  * `SITE_SEARCH`: `SOLUTION_TYPE_SEARCH` is automatically enrolled. Other
-                    solutions cannot be enrolled.
+                   * `MEDIA`: `SOLUTION_TYPE_RECOMMENDATION` and `SOLUTION_TYPE_SEARCH`.
+                   * `SITE_SEARCH`: `SOLUTION_TYPE_SEARCH` is automatically enrolled. Other
+                     solutions cannot be enrolled.
                 items:
                   type: string
                 type: array
               workspaceConfig:
                 description: Config to store data store type configuration for workspace
-                  data. This must be set when [DataStore.content_config][google.cloud.discoveryengine.v1.DataStore.content_config]
-                  is set as [DataStore.ContentConfig.GOOGLE_WORKSPACE][google.cloud.discoveryengine.v1.DataStore.ContentConfig.GOOGLE_WORKSPACE].
+                  data. This must be set when [DataStore.content_config][google.cloud.discoveryengine.v1alpha.DataStore.content_config]
+                  is set as [DataStore.ContentConfig.GOOGLE_WORKSPACE][google.cloud.discoveryengine.v1alpha.DataStore.ContentConfig.GOOGLE_WORKSPACE].
                 properties:
                   dasherCustomerID:
                     description: Obfuscated Dasher customer ID.
                     type: string
-                  superAdminEmailAddress:
```

Comment:
Ouch.  Does this mean that v1alpha1 is not a super-set of v1?

Reply:
Right, v1 and v1alpha are different, even the v1alpha client-go and the actual v1alpha server are different (server has more fields).